lib/kafka/ssl_socket_with_timeout.rb

```diff
@@ -163,6 +163,10 @@ def closed?
       @tcp_socket.closed? || @ssl_socket.closed?
     end
 
+    def to_io
```

Comment:
True. If only I could afford an SSL-enabled Kafka cluster! 😂

I think I'll hold off on merging this until I've at least had time to test that the current master works as expected in production, so as to not introduce too many new moving parts at a time.